rskj-core/src/test/java/co/rsk/peg/utils/BridgeEventLoggerImplTest.java

```diff
@@ -591,29 +585,37 @@ void logPegoutTransactionCreated_invalidBtcTxHashOrOutpointValues_shouldFail(Sha
     /**********************************
      *  -------     UTILS     ------- *
      *********************************/
-    private static void assertEvent(List<LogInfo> logs, int index, CallTransaction.Function event, Object[] topics, Object[] params) {
-        final LogInfo log = logs.get(index);
-        assertEquals(LogInfo.byteArrayToList(event.encodeEventTopics(topics)), log.getTopics());
-        assertArrayEquals(event.encodeEventData(params), log.getData());
+    private void assertEvent(CallTransaction.Function event, Object[] topics, Object[] data) {
+        LogInfo log = eventLogs.get(0);
+
+        byte[][] expectedEventTopicsSerialized = event.encodeEventTopics(topics);
+        List<DataWord> expectedEventTopics = LogInfo.byteArrayToList(expectedEventTopicsSerialized);
+        assertEquals(expectedEventTopics, log.getTopics());
+        
+        byte[] expectedEventData = event.encodeEventData(data);
+        assertArrayEquals(expectedEventData, log.getData());
     }
 
-    private void assertTopics(int topics, List<LogInfo> logs) {
-        assertEquals(topics, logs.get(0).getTopics().size());
+    private void assertTopics(int expectedTopicsSize) {
+        int topicsSize = eventLogs.get(0).getTopics().size();
+
+        assertEquals(expectedTopicsSize, topicsSize);
     }
 
-    private void commonAssertLogs(List<LogInfo> logs) {
```

Comment:
`logs` argument was always `eventLogs` global variable (what makes sense since we are resetting that variable in every test for unit testing properly).
So, the argument was removed since it was no really needed.